rascaline/src/math/mod.rs

```diff
@@ -1,9 +1,12 @@
 mod gamma;
-pub use self::gamma::{gamma, ln_gamma};
+pub use self::gamma::{EULER, gamma, ln_gamma, digamma};
```

Comment:
The EULER constant could be defined in this module directly, and imported in gamma and hyp2f1 as needed